modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -270,10 +278,23 @@ const EditorManager: EditorManager = {
   },
 
   /**
-   * Overrides the default options for editor instances.
+   * Overrides the default options for editor instances. The <code>overrideDefaults</code> method replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
+   * <br /><br />
+   * When using the Tiny Cloud, some of these defaults are required for the cloud-based editor to function.
+   * <br /><br />
+   * Therefore, when using <code>overrideDefaults</code> with the cloud-based editor, combine the previous defaults, <code>tinymce.defaultOptions</code>, with the new options that you want to integrate.
```

Comment:
A Socratic question:

Is _combine the previous defaults_ an instruction or a consequence?

To explain, one conceptual rendering of the current structure is as follows:

> _[state] has [requirements]_
> 
> _Therefore, when [state] + [thing], [instruction or consequence?], plus [thing] to create [state + more]._

If the phrase in question is an instruction, I’d argue for editing thus:
```suggestion
   * Overrides the default options for editor instances. The <code>overrideDefaults</code> method replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
   * <br /><br />
   * When using the Tiny Cloud, some of these defaults are required for the cloud-based editor to function.
   * <br /><br />
   * Therefore, when using <code>overrideDefaults</code> with the cloud-based editor, you must combine the previous defaults, <code>tinymce.defaultOptions</code>, with the new options that you want to integrate.
```

If, OTOH, the phrase in question is an expectation, I’d argue for editing thus:
```suggestion
   * Overrides the default options for editor instances. The <code>overrideDefaults</code> method replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
   * <br /><br />
   * When using the Tiny Cloud, some of these defaults are required for the cloud-based editor to function.
   * <br /><br />
   * Therefore, when using <code>overrideDefaults</code> with the cloud-based editor, code>overrideDefaults</code> combines the previous defaults, <code>tinymce.defaultOptions</code>, with the new options that you want to integrate.
```

Reply:
I'd say it's an instruction rather than a consequence, but I'll try to improve the wording without using a pronoun.